jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -2892,10 +2879,8 @@ public String toGenericString() {
 	 * IDENTITY shares the same bit as SYNCHRONIZED. VALUE shares the same bit as VOLATILE.
```

Comment:
Last comment nitpick is the mention on VALUE here.

Reply:
Removed the `VALUE` reference.